spec/js-api/value/index.d.ts

```diff
@@ -34,6 +34,8 @@ export const sassNull: Value;
  * > Sass value it represents.
  */
 export abstract class Value implements ValueObject {
+  protected constructor();
```

Comment:
Why is this needed?

Reply:
Typedoc doesn't know to automatically treat abstract constructors as protected. You're right that this seems like an oversight, though—I've filed https://github.com/TypeStrong/typedoc/issues/1729 to track the issue.